cmd/wctl/main.go

```diff
@@ -24,18 +24,20 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"fmt"
-	"github.com/perlin-network/wavelet/sys"
-	"github.com/perlin-network/wavelet/wctl"
-	"github.com/pkg/errors"
-	"gopkg.in/urfave/cli.v1"
 	"io/ioutil"
 	"os"
 	"sort"
 	"strconv"
 	"time"
+
+	"github.com/perlin-network/wavelet"
+	"github.com/perlin-network/wavelet/sys"
+	"github.com/perlin-network/wavelet/wctl"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
 )
 
-var tagConversion = map[string]byte{
+var tagConversion = map[string]sys.Tag{
```

Comment:
It would be better to either remove this map, or move it towards the `sys` pkg and keep the string naming consistent.